package.json

```
@@ -102,6 +102,12 @@
                     "default": "C:/Program Files/AutoHotkey/AutoHotkeyU64.exe",
                     "description": "Path to the AHK runner."
                 },
+                "ahk++.formatter.allowedNumberOfEmptyLines": {
+                    "type": "integer",
+                    "default": 1,
+                    "minimum": 1,
```

If we're going to make it customizable, we should make it fully customizable!

- 0: as expected, no empty lines allowed at all. Not sure why anyone would want this, but it may be good for helping to minify code
- -1: if value is set to -1, no empty lines are removed at all. This is easier than manually setting "1000" or some weirdly high number.

```suggestion
                    "minimum": -1,
```

---

Resolving, these can be separate work:
- #197 
- #198